testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/wsse/trust/WSTrustTestCaseElytronSecuritySetupTask.java

```diff
@@ -0,0 +1,136 @@
+package org.jboss.as.test.integration.ws.wsse.trust;
```

Comment:
@jimma This one is missing a copyright header.

Reply:
Fixed. Thanks , Darren.